docs/nightly/en/user-guide/ingest-data/for-iot/influxdb-line-protocol.md

```diff
@@ -1,11 +1,7 @@
-# InfluxDB Line Protocol
+# Influxdb Line Protocol
```

Comment:
we can keep InfluxDB for consistency

Reply:
hmmm...? there is nothing changed here, just modified the typo

Reply:
the type case here , we should keep `InfluxDB` where DB is in uppercase